ERROR project.version must not be blank

Running `jreleaser assemble` as per the README results in

```[ERROR] project.version must not be blank. Configure a value using the CLI yaml DSL, or define a System property jreleaser.project.version, or define a JRELEASER_PROJECT_VERSION environment variable, or define a key/value pair in /home/matthi/.jreleaser/config.properties with a key named JRELEASER_PROJECT_VERSION
JReleaser has not been properly configured.
project.version must not be blank. Configure a value using the CLI yaml DSL, or define a System property jreleaser.project.version, or define a JRELEASER_PROJECT_VERSION environment variable, or define a key/value pair in /home/matthi/.jreleaser/config.properties with a key named JRELEASER_PROJECT_VERSION```

--- a/README.adoc
+++ b/README.adoc
@@ -9,6 +9,9 @@ This repository shows how to configure link:https://github.com/features/actions[
  * assemble distributions with JReleaser
  * create rolling early-access releases with JReleaser on every push to `main` branch
 
+IMPORTANT: JReleaser requires 2 additional inputs that may be set as environment variables, `JRELEASER_PROJECT_VERSION` and `JRELEASER_GITHUB_TOKEN`. 
+More information about environment variable configuration may be found in the link:https://jreleaser.org/guide/latest/reference/environment.html[guide].
+
 There are two modes for packaging this application: classpath & modulepath.
 
 *classpath*

